tests/commonTemplates_test.go

```diff
@@ -208,6 +208,30 @@ var _ = Describe("Common templates", func() {
 	})
 
 	Context("resource change", func() {
+
+		Context("namespace change", func() {
+			AfterEach(func() {
```

Comment:
Please add:
```go
BeforeEach(func() {
	strategy.SkipSspUpdateTestsIfNeeded()
}
```

The test suite can be configured to skip any tests that change the SSP resource. This is useful if some other operator watches the SSP and it could revert any changes in the middle of the test.